tests/everest/test_everlint.py

```diff
@@ -13,6 +13,26 @@
 from tests.everest.utils import relpath
 
 
+@pytest.fixture
+def min_config():
+    yield yaml.safe_load(
+        dedent("""
+    model: {"realizations": [0]}
+    controls:
+      -
+        name: my_control
+        type: well_control
+        min: 0
+        max: 0.1
+        variables:
+          - { name: test, initial_guess: 0.1 }
+    objective_functions:
+      - {name: my_objective}
+    config_path: .
+    """)
+    )
+
+
```

Comment:
This thing is already in `conftest.py`, I think? Does it need to be here? Is it duplicated now?

Reply:
Ah, I completely missed that, we don't need it indeed :)! Thanks for spotting.